### PR TITLE
Spike/testing

### PR DESCRIPTION
**Pull request by aidan-starke**
## Description

 - Add more test cases

#### Tests For:
 - `fetchUnclaimedEventProof`
 - `fetchUnclaimedWithdrawals`
 - `fetchDepositRelayerStatus`, `ensureRelayerDepositDone`
 - `Balance`
 - `fetchBridgeDepositStatus`, `fetchBridgeWithdrawStatus`
 - `fetchMetaMaskBalance`
 - `getAddLiquidityExtrinsic`, `getRemoveLiquidityExtrinsic`
 - `getBuyAssetExtrinsic`, `getSellAssetExtrinsic`
 - `getExpiryString`
 - `getPegWithdrawExtrinsic`
 - `isEthereumAddress`, `isCENNZAddress`
 - `signAndSendTx`
 - `sendDepositRequest`
 - `sendWithdrawEthereumRequest`
 - `getTokenLogo`

### CODE REVIEW

File: tests/getAddLiquidityExtrinsic.test.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+		const extrinsic = await getAddLiquidityExtrinsic(
+			api,
+			exchangeInfo,
+			cennzAsset.assetId,
+			tradeAssetValue,
+			coreAssetValue,
+			slippage
+		);
+
```
Comment (ken-centrality) [inline tests/getAddLiquidityExtrinsic.test.ts:27]: the more effective way to test the extrinsic is to inspect its `.args` property and ensure the variables are constructed as expected

Screen Shot 2022-04-29 at 13 31 58

File: tests/getBuyAssetExtrinsic.test.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+		const extrinsic = getBuyAssetExtrinsic(
+			api,
+			cennzAsset.assetId,
+			exchangeAssetValue,
+			cpayAsset.assetId,
+			receivedAssetValue,
+			slippage
+		);
+
```
Comment (ken-centrality) [inline tests/getBuyAssetExtrinsic.test.ts:21]: same, use `args` inspection test

File: tests/getPegWithdrawExtrinsic.test.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+describe("getPegWithdrawExtrinsic", () => {
+	it("returns expected extrinsic", () => {
+		const transferAssetValue = new Balance(10, cennzAsset);
+		const extrinsic = getPegWithdrawExtrinsic(
+			api,
+			cennzAsset.assetId,
+			transferAssetValue,
+			testingAccount
+		);
```
Comment (ken-centrality) [inline tests/getPegWithdrawExtrinsic.test.ts:16]: same, use `args` inspection test

File: tests/getRemoveLiquidityExtrinsic.test.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+		const extrinsic = await getRemoveLiquidityExtrinsic(
+			api,
+			exchangeInfo,
+			cennzAsset,
+			tradeAssetValue,
+			coreAssetValue,
+			slippage
+		);
+
```
Comment (ken-centrality) [inline tests/getRemoveLiquidityExtrinsic.test.ts:27]: same, use `args` inspection test

File: tests/getSellAssetExtrinsic.test.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+		const extrinsic = getSellAssetExtrinsic(
+			api,
+			cennzAsset.assetId,
+			exchangeAssetValue,
+			cpayAsset.assetId,
+			receivedAssetValue,
+			slippage
+		);
+
```
Comment (ken-centrality) [inline tests/getSellAssetExtrinsic.test.ts:21]: same, use `args` inspection test

**Review by ken-centrality — CHANGES_REQUESTED** [review]

File: tests/getAddLiquidityExtrinsic.test.ts (hunk 1 above)
Comment (aidan-starke) [reply tests/getAddLiquidityExtrinsic.test.ts:27]: not sure I can, all I've got is

```
console.log
    extrinsic Submittable {
      createdAtHash: undefined,
      initialU8aLength: undefined,
      signaturePayloadOptions: {
        blockHash: undefined,
        genesisHash: undefined,
        method: undefined,
        specVersion: undefined,
        nonce: undefined,
        runtimeVersion: undefined,
        transactionPayment: undefined,
        transactionVersion: undefined
      }
    }
```

**Review by ken-centrality — APPROVED** [review]

**Pull request merged**